first session/assignment2/totulist.css

```
@@ -0,0 +1,150 @@
+
+
+ body {
+    font-family: '나눔고딕', sans-serif;
+    background: linear-gradient(to right, #74ebd5, #acb6e5);
+    display: flex;
+
+    flex-direction: column;
+    
+    justify-content: flex-start;
+    align-items: center;
+    height: 100vh;
+    margin: 0;
+    padding: 20px;
+    overflow: auto;
+}
+
+.todo-container {
+    background: white;
+    padding: 20px;
+    border-radius: 8px;
+    box-shadow: 0 0 10px rgba(0,0,0,0.1);
+}
+
+.not-container {
+   
+
+    background: white;
+    padding: 20px;
+    border-radius: 8px;
+    box-shadow: 0 0 10px rgba(0,0,0,0.1);
+}
+
+.done-container{
+    margin-left: 150px;
+    position: center;
+    background: white;
```

통일성을 위해 배경색만 지정할때는 background-color를 사용해줍시당!!
```suggestion
    background-color: white;
```